README.md

```diff
@@ -10,10 +10,9 @@ You need to have npm installed on your local machine.
 
 clone the repo
 
-https://github.com/david63011/todo-Project.git
-
+https://github.com/david63011/Leaderboard.git
 # Live app
-https://david63011.github.io/todo-Project/dist/
+not yet...
 
 # Install
 
```

Comment:
- [ ] Kindly make sure you use the correct [MICROVERSE README](https://github.com/microverseinc/readme-template/blob/master/README.md) template by adding and filling in all the important sections(e.g.: good project description, correctly fill the `## Authors` section as from the template,...). I suggest you take the copy from **the [template repo](https://github.com/microverseinc/readme-template/blob/master/README.md)**, update it according to your project info and to make it easy you can use this **[example](https://github.com/tresorsawasawa/Leaderboard_App/blob/development/README.md)** to help you fill correctly all the sections.

Reply:
ok I improved it